app/services/catalog/create_approval_request.rb

```diff
@@ -23,7 +23,7 @@ def process
     def submit_approval_requests(order_item)
       workflows(order_item.portfolio_item).each do |workflow|
         response = Approval.call { |api| api.create_request(workflow, request_body_from(order_item)) }
-        order_item.approval_requests << create_approval_request(response)
+        order_item.approval_requests << create_approval_request(response, order_item)
```

Comment:
The `<<` operator already knows about the `order_item_id` by reference. If we were to set the `tenant_id` from the associated `order_item` in a callback we could further remove this change.